kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/embedder/MavenProjectLoader.java

```diff
@@ -38,19 +45,22 @@
 public class MavenProjectLoader {
 
     public static final String FORCE_OFFLINE = "kie.maven.offline.force";
+    public static final String RESOLVE_TRANSITVE = "kie.maven.offline.transitive.resolve";
+
     protected static boolean IS_FORCE_OFFLINE = Boolean.valueOf(System.getProperty(FORCE_OFFLINE, "false"));
+    protected static boolean IS_RESOLVE_TRANSITIVE = Boolean.valueOf(System.getProperty(RESOLVE_TRANSITVE, "false"));
 
     private static final Logger log = LoggerFactory.getLogger(MavenProjectLoader.class);
 
-    private static final String DUMMY_POM =
-                    "    <project xmlns=\"http://maven.apache.org/POM/4.0.0\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" +
-                    "      xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\">\n" +
-                    "      <modelVersion>4.0.0</modelVersion>\n" +
-                    "     \n" +
-                    "      <groupId>myGroupId</groupId>\n" +
-                    "      <artifactId>myArtifactId</artifactId>\n" +
-                    "      <version>1.0-SNAPSHOT</version>\n" +
-                    "    </project>";
+    private static final String DUMMY_POM = 
+                                            "    <project xmlns=\"http://maven.apache.org/POM/4.0.0\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" +
+                                            "      xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\">\n" +
+                                            "      <modelVersion>4.0.0</modelVersion>\n" +
+                                            "     \n" +
+                                            "      <groupId>myGroupId</groupId>\n" +
+                                            "      <artifactId>myArtifactId</artifactId>\n" +
+                                            "      <version>1.0-SNAPSHOT</version>\n" +
+                                            "    </project>";
```

Comment:
is this only format?

Reply:
only format

Reply:
so please, revert all format changes. so we have a clear diff to review